distribution/server-quarkus/pom.xml

```diff
@@ -0,0 +1,176 @@
+<!--
+  ~ Copyright 2016 Red Hat, Inc. and/or its affiliates
+  ~ and other contributors as indicated by the @author tags.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>keycloak-distribution-parent</artifactId>
+        <groupId>org.keycloak</groupId>
+        <version>8.0.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>keycloak-server-quarkus</artifactId>
+    <packaging>pom</packaging>
+    <name>Keycloak Quarkus Server Distribution</name>
+    <description/>
+
+    <dependencies>
+        <dependency>
```

Comment:
Should not depend on keycloak-server-feature-pack (as that will eventually be removed, granted it is a long time from now). Rather add licenses in the same that it does.

Reply:
Removed as discussed, separate ticket created to add licenses at a later date